libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -31,11 +31,8 @@
 #include <GCS_MAVLink/GCS.h>
 #include <AP_SerialManager/AP_SerialManager.h>
 #include <AP_Logger/AP_Logger.h>
-<<<<<<< HEAD
```

Comment:
this needs some git squashing, we don't want conflict markers in the history

Reply:
fixed